test/parse/test_scope.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+from unittest import TestCase
+
+from pytest import raises
+
+from flash_patcher.exception.error_manager import ErrorManager
+from flash_patcher.parse.scope import Scope
+
+class ScopeSpec (TestCase):
+
```

Comment:
Unnecessary whitespace?

Reply:
This seems to be standard across our test cases, will keep as is